tests/integration.rs

```diff
@@ -54,5 +54,136 @@ mod integration_tests {
         }
     }
 
+    #[cfg(unix)]
+    mod http_tests_unix {
```

Comment:
This is because I am using a UNIX-specific interruption to stop the HTTP server when the test is done. 

See this line
`child.interrupt().expect("Error interrupting child");`

In Windows, the signal to interrupt the HTTP server, or a process in general, is different from UNIX. 

With that said, I am intended to write HTTP integration tests that work on the Windows platform too, but not in this PR.